### ISSUE TITLE
Create Screencast(s) for installation && major milestones

### ISSUE THREAD

**Issue by Kentoseth** [OriginProtocol/platform#20]
It would be a nice feature/video where there is a walk-through of basic installation and usage of `origin.js`.

Maybe building a series of videos where each major milestone is shared in a video for both beginners and others working on other parts of `origin.js` to get a full summary of what is new.

**Comment by micahalcorn** [comment]
https://www.useloom.com/share/41fa20580f4c423e851ae014085aacc4

**Comment by wanderingstan** [comment]
This is a *great* idea for our recent push to improve new-developer onboarding experience. 

@micahalcorn do you recommend loom over e.g. basic youtube videos?

**Comment by micahalcorn** [comment]
Nah, something like YouTube would probably be better for discoverability. Loom is just amazing for quick and easy screencast sharing. 💻

**Comment by allisonphillips** [comment]
It looks like you should be able to make a loom video, download it, and then post it on YouTube
http://support.useloom.com/faqs-and-troubleshooting/how-to-download-a-video-for-youtube-vimeo-dropbox-etc

**Comment by micahalcorn** [comment]
Nice, thanks @allisonphillips 😄
